doc/configuration.rst

```diff
@@ -74,6 +74,26 @@ Globals Command-line options
 
   * ``--consul_ssl_verify``: If set to false, disable Consul certificate checking (true by default is ssl enabled).
 
+.. _option_terraform_plugins_dir_cmd:
+
+  * ``--terraform_plugins_dir``: Specify the directory where to find Terraform pre-installed providers plugins. If not specified, required plugins will be downloaded during deployment. See https://www.terraform.io/guides/running-terraform-in-automation.html#pre-installed-plugins for more information.
+
+.. _option_terraform_aws_plugin_version_constraint_cmd:
+
+  * ``--terraform_aws_plugin_version_constraint``: Specify the Terraform AWS plugin version constraint. Default one compatible with our source code is ``"~> 1.36"``. If you choose another, it's at your own risk. See https://www.terraform.io/docs/configuration/providers.html#provider-versions for more information.
```

Comment:
Same as previous comment we may be able to automate generation of those versions by looking at the versions.yaml file. 
versions.yaml should be the only source of truth.